deploy/README.md

```diff
@@ -1,27 +1,121 @@
 # Overview of Infrastructure deployment and configuration
 
-The deployment involves the following steps outlined below:
+We are using bicep templates to deploy the infrastructure resources.
+We will be using following azure services:
+- Azure Batch Account
+- Azure Synapse Analytics
+- Azure Keyvault
+- Azure Storage Account
+- Azure Networking
+- Azure Log Analytics
 
-No | Step | Duration (approx.) | Required / Optional
----|------|----------|---------------------
-1 | [Preparing to execute the script](./deploy-infrastructure.md#preparing-to-execute-the-script) | 1 minute | required
-2 | [Infrastructure Deployment](./deploy-infrastructure.md#infrastructure-deployment) | 10 minutes | required
-3 | [Verifying infrastructure resources](./deploy-infrastructure.md#verifying-infrastructure-resources) | 5 minutes | required
+# Prerequisites
 
-Follow the [document](./deploy-infrastructure.md) to understand and execute the infrastructure deployment process.
+The deployment script uses following tools, please follow the links provided to install the suggested tools on your computer using which you would execute the script.
 
-## Executing the synapse pipeline
+- [git](https://github.com/git-guides/install-git) 
+- [az cli](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [jq](https://stedolan.github.io/jq/download/)
+- The scripts are executed on bash shell, so if using a computer with windows based operating system, install a [WSL](https://docs.microsoft.com/windows/wsl/about) environment to execute the script.
+- Install bicep on az cli using command `az bicep install`.
+- The bicep templates have been written to adhere to the syntax and rules for bicep version >= 0.8.9. Please check your bicep version using `az bicep version` if you run into bicep related errors.
+- The user performing the deployment of the bicep template and the associated scripts should have `Owner` role assigned at the subscription to which the resources are being deployed.
+- This solution assumes no interference from Policies deployed to your tenant preventing resources from being deployed. 
+- Get the repository to find the scripts. Clone the repository using following command.
 
-We have developed a sample Azure Synapse Pipeline demostrating the use of custom-vision ai-model to process a sample image.
+```bash
+git clone git@github.com:Azure/Azure-Orbital-Analytics-Samples.git
+```
 
-Follow the [document](./using-pipeline.md#executing-the-pipeline) to configure and execute the pipeline.
+Alternatively, one can use Azure Cloud Bash to deploy this sample solution in their Azure subscription.
+
+## Infrastructure Deployment
+
+Before executing the script one would need to login to azure using `az` cli and set the subscription in which they want to provision the resources.
+
+```bash
+az login
+az account set -s <subscription_id>
+```
+
+To install, configure and package the infrastructure for dependencies one can execute the script [setup.sh](./setup.sh)
```

Comment:
Can we update lines from 41 to 54 to something like below:

The following command will install, configure and generate the custom vision model package.

```bash
./deploy/setup.sh <environmentCode> <location>
```
- `environmentCode` used as a prefix for Azure resource names. It allows only alpha numeric(no special characters) and must be between 3 and 8 characters.
- `location` is a valid Azure region.

For eg.
```bash
./deploy/setup.sh aoi eastus
```